ckanext/spatial/plugin.py

```diff
@@ -5,13 +5,12 @@
 
 from pylons import config
 
-import shapely
+
 
 from ckan import plugins as p
 
-from ckan.lib.search import SearchError, PackageSearchQuery
 from ckan.lib.helpers import json
-
+import shapely.geometry
```

Comment:
I don't think this import is required.  It was moved to `before_index()` in an upstream commit.